src/tasks/__init__.py

```diff
@@ -16,7 +16,7 @@
 ##
 # Task lists for handling as a group; these names correspond to the keys in 
 # the task registry.
-ALL_GLUE_TASKS = ['sst', 'cola', 'mrpc', 'qqp', 'sts-b',
+ALL_GLUE_TASKS = ['sst', 'cola', 'cola-in-domain', 'cola-out-domain', 'mrpc', 'qqp', 'sts-b',
```

Comment:
Let's not do this, at least for now. The GLUE version of CoLA is the version with only a single evaluation set. If people want the full CoLA eval, they can explicitly add it to the task list.